net/grpc/gateway/docker/prereqs/Dockerfile

```diff
@@ -19,22 +19,24 @@
 # Stage 1: Fetch binaries
 ######################################
 # node:... Docker image is based on buildpack-deps:stretch
-FROM buildpack-deps:stretch AS prepare
+FROM buildpack-deps:bullseye AS prepare
```

Comment:
This change was made to match base image versions with the requirement to use `node16:bullseye` below